src/libMessage/Messenger.cpp

```diff
@@ -1390,6 +1403,34 @@ void DSBlockHeaderToProtobuf(const DSBlockHeader& dsBlockHeader,
       SerializableToProtobufByteArray(winner.second,
                                       *powdswinner->mutable_val());
     }
+    // TODO @Chetan : serialize govdata in dsblock
+    // double check on serialization and deserialization part
+    ZilliqaMessage::ProtoDSBlock::DSBlockHeader::Proposal* protoproposal;
+    for (const auto& govProposal : dsBlockHeader.GetGovProposalMap()) {
+      protoproposal = protoDSBlockHeader.add_proposals();
+      protoproposal->set_proposalid(govProposal.first);
+      for (const auto& vote : govProposal.second.first) {
+        LOG_GENERAL(INFO, "Governance: serialization DS nodes govProposalId: "
```

Comment:
Avoid long log messages with a lot of colons used for both labeling ("Governance:") and separating key-value pairs (quite hard to read).

Also, aim for consistency in log formats (for example, "govVoteValue:" has no trailing space but "voteCount: " has).

You might want to go with something like this instead (note: no spaces around `=`):
```
[Gov][DS] Proposal=<ID> Value=<value> Count=<count>
[Gov][Shards] Proposal=<ID> Value=<value> Count=<count>
```

Reply:
yeah. I changed the log lines as suggested,
But this log , i will remove later on.I added them to verify if serialization and de-serialization works fine.